eq-author/src/App/section/index.js

```diff
@@ -16,4 +17,15 @@ export default [
     path="/q/:questionnaireId/section/:sectionId/preview"
     component={Preview}
   />,
+  <Route
+    key="section-logic"
+    path="/q/:questionnaireId/section/:sectionId/logic"
+  >
+    <Redirect to="display" />
+  </Route>,
+  <Route
+    key="section-logic-display"
+    path="/q/:questionnaireId/section/:sectionId/display"
+    component={Display}
+  />,
```

Comment:
You'll need to do something here to stop people navigating to the logic page via a URL on V2.

Similar to the above, you can use the `enableOn(['hub'])` and `disableOn(['hub'])` functions to help.